spec/models/generic_object_spec.rb

```diff
@@ -231,6 +231,21 @@
     end
   end
 
+  describe 'property methods without user set' do
+    it 'will set the current user' do
+      workspace = double("MiqAeWorkspaceRuntime", :root => {"method_result" => "result"})
+      User.current_user = user
```

Comment:
@jntullo 
Instead of setting the User.current_user could you use User.with_user_group(user, group) { spec commands }
akin to https://github.com/ManageIQ/manageiq/blob/master/spec/lib/rbac/filterer_spec.rb#L185